README.md

```diff
@@ -24,6 +24,11 @@ You can use the `docker-compose` to run the app and DB servers by executing the
 $ docker-compose up --build
 ```
 
+To run the secondary app and Servers for testing run:
+```shell 
+$ docker-compose -p datamapper -f Docker-compose-data-mapper.yml up -d
```

Comment:
nit: lower case on the file

nit: while `docker-compose` works, the recommended way to do it now is to use `docker compose ...` (compose is part of docker cmd) - [reference](https://docs.docker.com/compose/migrate/)